src/main.ts

```diff
@@ -21,100 +21,100 @@ export async function run(
-      )
-    }
+  const result = await runInCI(refs, api, options, git)
 
-    if (result.mode === 'standalone') {
-      const id = await uploadArtifact(
+  if (result.commentId != null) {
+    core.setOutput('comment-id', result.commentId)
+    core.info(`Commented on PR #${github.context.issue.number}`)
+  }
+
+  const diffFiles =
+    result.mode === 'standalone'
+      ? Object.values(result.files.diff ?? {})
+      : result.diffPath
+        ? [result.diffPath]
+        : []
+
+  if (diffFiles.length > 0) {
+    await uploadArtifact(artifact, createDiffArtifactName(), diffFiles, inputs)
+  }
+
+  if (result.mode === 'standalone') {
+    const id = await uploadArtifact(
+      artifact,
+      createReportArtifactName(),
+      Object.values(result.files.report),
+      inputs
+    )
+    core.setOutput('artifact-id', id)
+  } else {
+    for (const project of result.projects) {
+      await uploadArtifact(
         artifact,
-        createReportArtifactName(),
-        Object.values(result.files.report),
+        createReportArtifactName(project.name),
+        Object.values(project.files.report),
         inputs
       )
-      core.setOutput('artifact-id', id)
-    } else {
-      for (const project of result.projects) {
+      if (project.files.diff) {
         await uploadArtifact(
           artifact,
-          createReportArtifactName(project.name),
-          Object.values(project.files.report),
+          createDiffArtifactName(project.name),
+          Object.values(project.files.diff),
           inputs
         )
-        if (project.files.diff) {
-          await uploadArtifact(
-            artifact,
-            createDiffArtifactName(project.name),
-            Object.values(project.files.diff),
-            inputs
-          )
-        }
       }
     }
+  }
 
-    if (inputs.annotations) {
-      const issues =
-        result.mode === 'standalone'
-          ? (result.newIssues ?? [])
-          : result.projects.flatMap(project => project.newIssues ?? [])
-      if (issues.length > 0) {
-        core.info(
-          `Found ${issues.length} new issues, creating GitHub annotations`
-        )
-        createAnnotationsFromIssues(issues)
-      } else {
-        core.info('No new issues found, skipping GitHub annotations')
-      }
+  if (inputs.annotations) {
+    const issues =
+      result.mode === 'standalone'
+        ? (result.newIssues ?? [])
+        : result.projects.flatMap(project => project.newIssues ?? [])
+    if (issues.length > 0) {
+      core.info(
+        `Found ${issues.length} new issues, creating GitHub annotations`
+      )
+      createAnnotationsFromIssues(issues)
+    } else {
+      core.info('No new issues found, skipping GitHub annotations')
     }
-  } catch (error) {
-    const errorMessage = error instanceof Error ? error.message : `${error}`
-    core.error(errorMessage)
-    core.setFailed(errorMessage)
-    return
   }
-
-  core.info(`${LOG_PREFIX} Finished running successfully`)
```

Comment:
Was this removed on purpose? I thought this log was quite useful.

Reply:
Actually no, I'll add it back